WordPress/src/main/res/layout/my_site_card_toolbar.xml

```diff
@@ -33,9 +33,11 @@
         android:contentDescription="@string/content_description_more"
         android:focusable="true"
         android:src="@drawable/ic_more_vert_white_24dp"
+        android:visibility="gone"
```

Comment:
@ashiagr - Chris wanted us to keep the space for the more menu even when it isn't visible, so the padding remains constant (since we are alining the title with the more menu). Should we flip this from `gone` to `invisible`?

Reply:
Flipped in c2c96d1